src/test/java/core/basesyntax/service/impl/ValidatorServiceImplTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ValidatorService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ValidatorServiceImplTest {
+    private static ValidatorService validator;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
```

Comment:
why throws?

Reply:
Forget to delete after generation, my last commit was after midnight.

Reply:
Done